plugins/newsdownloader.koplugin/main.lua

```diff
@@ -105,6 +106,21 @@ function NewsDownloader:addToMainMenu(menu_items)
                 keep_menu_open = true,
                 callback = function() self:removeNewsButKeepFeedConfig() end,
             },
+            {
+                text = _("Force skip images download"),
```

Comment:
@NiLuJe Do you have any thoughts? I'm not overly fond of this string. (Setting aside that `Force skip image download` would be the more idiomatic form of it.)

Reply:
`Never download images`?

Reply:
Probably applies to a bunch of similar strings up the chain, then ;).

(and/or variable names if someone wants to go the extra mile and it makes sense re: the rest of the code).

Reply:
Yes, my thoughts exactly (also see [here](https://github.com/koreader/koreader/pull/6003/commits/2bd831205941b6b7c5cc2310772890b4565d2e7b#r398161735) because I mistakenly thought the commits were distinct units). :-)
```suggestion
                text = _("Never download images"),
```

Reply:
And yes, the variable name would also be much clearer as never_download_images

Reply:
Guys IMHO  "never" means something that will burn down function (or even hardware? (if I would have been non-tech) :p) responsible for downloading images.
I would NEVER click that (if I didn't write that function):p
I'm serious. 

This is "temporarily disable", you can quickly reverse it and its gets back to normal downloading images not forever! 

Please rethink once again you request

Reply:
How about " temporary skip images"?

Reply:
That's because I overlooked that you made it scary, and it also explains the weird popup. ;-)

<img src=https://user-images.githubusercontent.com/202757/77622024-d70f1700-6f3d-11ea-83c1-8c6d259aac99.png width=45%>

Should be like this instead:
```diff
diff --git a/plugins/newsdownloader.koplugin/main.lua b/plugins/newsdownloader.koplugin/main.lua
index 5f3921b1..debbaa64 100644
--- a/plugins/newsdownloader.koplugin/main.lua
+++ b/plugins/newsdownloader.koplugin/main.lua
@@ -107,18 +107,20 @@ function NewsDownloader:addToMainMenu(menu_items)
                 callback = function() self:removeNewsButKeepFeedConfig() end,
             },
             {
-                text = _("Force skip images download"),
+                text = _("Never download images"),
                 keep_menu_open = true,
+                checked_func = function()
+                    -- Not sure this should all be quite this local all over the file ;-)
+                    local news_downloader_settings = LuaSettings:open(("%s/%s"):format(DataStorage:getSettingsDir(), news_downloader_config_file))
+                    local skip = news_downloader_settings:readSetting(force_skip_images_download_config_name)
+                    return skip
+                end,
                 callback = function()
                     local news_downloader_settings = LuaSettings:open(("%s/%s"):format(DataStorage:getSettingsDir(), news_downloader_config_file))
                     local skip = news_downloader_settings:readSetting(force_skip_images_download_config_name) or false;
                     logger.warn('NewsDownloader: previous force_skip_images_download: ', skip)
                     news_downloader_settings:saveSetting(force_skip_images_download_config_name, not skip)
                     news_downloader_settings:flush()
-                    UIManager:show(InfoMessage:new{
-                        text = T(_("Force skip images download set to %1"),
-                        string.format("%s", not skip))
-                    })
                 end,
             },
             {
```

<img src=https://user-images.githubusercontent.com/202757/77622871-5cdf9200-6f3f-11ea-9e34-4783f4ec488f.png width=45%>